packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -91,38 +94,120 @@
                 border-right: 1px solid #ededed;
             }
         }
+
+        @include mobile {
+            width: 110vw;
```

Comment:
This will exceed the width of the page, is this intentional?

Reply:
Yeap, this is for the horizontal scroll of stats.

Reply:
Hmm, how about adding a `max-width: 100vw` on this and then setting the child's overflow (through  ThemedScrollbars)? You could flex the two children (the statistics and the info icon) next to each other.

Reply:
I've incorporated this change, this was a great suggestion, thank you!